lib/sentry_dart_plugin.dart

```diff
@@ -75,15 +78,86 @@ class SentryDartPlugin {
       Log.info('includeSources is disabled, not uploading sources.');
     }
 
-    params.add(_configuration.buildFilesFolder);
-
     _addWait(params);
 
-    await _executeAndLog('Failed to upload symbols', params);
+    final debugSymbolPaths = _enumerateDebugSymbolPaths();
+    final fs = injector.get<FileSystem>();
+    await for (final path in debugSymbolPaths) {
+      if (await fs.directory(path).exists() || await fs.file(path).exists()) {
+        await _executeAndLog('Failed to upload symbols', [...params, path]);
+      }
+    }
+
+    for (final path in await _enumerateSymbolFiles()) {
+      await _executeAndLog('Failed to upload symbols', [...params, path]);
+    }
 
     Log.taskCompleted(taskName);
   }
 
+  Stream<String> _enumerateDebugSymbolPaths() async* {
```

Comment:
Any particular reason why this is a stream when we are "just" returning a list of directories?

Reply:
none, just convenient syntax